fairlearn/metrics/extra_metrics.py

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import sklearn.metrics as skm
+
+
+def specificity_score(y_true, y_pred, sample_weight=None):
```

Comment:
I'm slightly concerned that we should be providing the richness of the `scikit-learn` metrics here - see for example:
https://scikit-learn.org/stable/modules/generated/sklearn.metrics.recall_score.html
They provide a lot of support for multiclass metrics